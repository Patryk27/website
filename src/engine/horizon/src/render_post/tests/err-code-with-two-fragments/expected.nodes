Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "code",
                span: Span {
                    start: 1,
                    end: 5,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  ",
                        span: Span {
                            start: 6,
                            end: 9,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n    fn main() {\n        println!(\"hello\");\n    }\n  ",
                        span: Span {
                            start: 13,
                            end: 66,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n\n  ",
                        span: Span {
                            start: 68,
                            end: 72,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n    fn main() {\n        println!(\"world\");\n    }\n  ",
                        span: Span {
                            start: 76,
                            end: 129,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n",
                        span: Span {
                            start: 131,
                            end: 132,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 0,
            end: 139,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 139,
            end: 140,
        },
    },
)