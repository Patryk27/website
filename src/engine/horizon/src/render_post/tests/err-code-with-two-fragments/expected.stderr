[31merror:[0m unexpected node
    â•­â”€[ <unknown>:8:7 ]
    â”‚
  8 â”‚ â•­â”€â–¶   <!--
    â”† â”†   
 12 â”‚ â”œâ”€â–¶   -->
    â”‚ â”‚           
    â”‚ â•°â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ unexpected node
â”€â”€â”€â”€â•¯

compilation failed
